recipes/_configure_server.rb

```diff
@@ -73,8 +73,11 @@
     k.start_with?('_cerner_splunk') || SLAVE_ONLY_CONFIGS.include?(k)
   end
 
+  pass = settings.delete('pass4SymmKey')
+
   server_stanzas['clustering'] = settings
   server_stanzas['clustering']['mode'] = 'master'
+  server_stanzas['clustering']['pass4SymmKey'] = CernerSplunk::ConfTemplate.compose encrypt_password, CernerSplunk::ConfTemplate::Value.constant(value: pass) if pass
```

Comment:
I'm guessing it would fall back to the general stanza rather than using the clustering stanza for indexer discovery, but I'll see if I can test that scenario locally to verify.

Reply:
Tested by setting a pass4SymmKey under the settings hash in cluster-indexer-discovery.json and cluster-vagrant.json.  Spun up `vagrant up c1_master c1_slave1 c1_slave2 c1_search f_default` and verified on c1_search that logs from all 5 nodes were showing up as expected.  If the key isn't set under indexer_discovery, it falls back to general as expected, not the clustering stanza.